requirements.txt

```diff
@@ -1,2 +1,3 @@
 requests
-pytz
\ No newline at end of file
+pytz
+ovos_workshop~=0.0.10
```

Comment:
please use ~=0.0, >=0.0.10
seems to work better when handling requirements with pip installs